modules/doc/content/newsletter/2025/2025_04.md

```diff
@@ -13,6 +8,10 @@ update your MacOS version, please consider using the [Docker-based installation
 
 ## MOOSE Improvements
 
+### MFEM backend support
+
+MOOSE now has base support for finite element computations using [MFEM](mfem.org). This support was developed by the United Kingdom Atomic Energy Authority (UKAEA) originally as part of their Platypus application, but that support has now been merged upstream into MOOSE. Those wishing to get familiar with use of the MFEM backend can look at [test cases](https://github.com/idaholab/moose/tree/next/test/tests/mfem) as well as the class documentation under the MFEM header on our [source index page](https://mooseframework.inl.gov/source/index.html).
```

Comment:
```suggestion
MOOSE now has base support for finite element computations using [MFEM](mfem.org). This support was developed by the United Kingdom Atomic Energy Authority (UKAEA), originally as part of their [Platypus application](https://github.com/aurora-multiphysics/platypus), but that support has now been merged upstream into MOOSE. Those wishing to get familiar with use of the MFEM backend can look at [test cases](https://github.com/idaholab/moose/tree/next/test/tests/mfem) as well as the class documentation under the MFEM header on our [source index page](https://mooseframework.inl.gov/source/index.html).
```

One grammar tweak, and I think it's good to add a link to their original application for both full attribution and context.